modules/builds-strategy-custom-environment-variables.adoc

```diff
@@ -23,12 +23,13 @@ customStrategy:
       value: "http://myproxy.net:5187/"
 
 ----
-
-ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
-Cluster administrators can also configure global build settings using Ansible.
-endif::[]
-
-You can also manage environment variables defined in the `BuildConfig` with the `oc set env` command.
++
+. To manage environment variables defined in the `BuildConfig`, enter the follwoing command:
```

Comment:
Attempted to clarify procedure. Might return to original since it's pretty far out of scope for this PR.